html/recipe.html

```diff
@@ -15,26 +15,27 @@
   <div class="grid">
```

Comment:
Hey should we remove this html file since we are just on one page? I suggest solely working off of index.html